specs/linux/kernel_keys.table

```diff
@@ -0,0 +1,23 @@
+table_name("kernel_keys")
+description("List of security data, authentication keys and encryption keys.")
+schema([
+    Column("serial_number", TEXT, "The serial key of the key."),
+    Column("flags", TEXT, "A set of flags describing the state of the key."),
+    Column("usage", BIGINT, "the number of threads and open file references that"
+                     "refer to this key."),
+    Column("timeout", TEXT, "The amount of time until the key will expire,"
+                     "expressed in human-readable form. The string perm here"
+                     "means that the key is permanent (no timeout).  The"
+                     "string expd means that the key has already expired."),
+    Column("permissions", TEXT, "The key permissions, expressed as four hexadecimal"
```

Comment:
It'll be interesting seeing if this field proves usable, of if we can improve it.